README.md

```diff
@@ -1,12 +1,47 @@
-# AWS IoT SigV4 Utility Library
+# AWS IoT SigV4 Library
 
 **Note** This library is currently under development.
 
-The AWS IoT SigV4 Library is a standalone utility for generating a signature and authorization header according to the specifications of the [Signature Version 4](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html) signing process. This utility is an optional addition to applications sending direct HTTP requests to AWS services requiring SigV4 authentication.
+The AWS IoT SigV4 Library is a standalone library for generating authorization header and signature according to the specifications of the [Signature Version 4](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html) signing process. Authorization header is the required for authentication when sending HTTP requests to AWS. This Library is an optional addition to applications sending direct HTTP requests to AWS services requiring SigV4 authentication. This library has no dependencies on any additional libraries other than the standard C library. This library is distributed under the MIT Open Source License.
```

Comment:
```suggestion
The AWS IoT SigV4 Library is a standalone library for generating authorization headers and signatures according to the specifications of the [Signature Version 4](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html) signing process. Authorization headers are required for authentication when sending HTTP requests to AWS. This library can optionally be used by applications sending direct HTTP requests to AWS services requiring SigV4 authentication. This library has no dependencies on any additional libraries other than the standard C library. This library is distributed under the MIT Open Source License.
```

Reply:
Suggest removing the IoT prefix

Reply:
Shall I remove it now or first discuss it with Tanmoy.